assets/prototype/domain/eventEditor/datetimes/dateCard/DateCard.tsx

```diff
@@ -51,17 +34,17 @@ const DateCard = ({ id }) => {
 		relation: 'tickets',
 	});
 
-	const startDate = moment(date.startDate).toDate() || PLUS_ONE_MONTH;
-	const endDate = moment(date.endDate).toDate() || PLUS_TWO_MONTHS;
-	const defaultRangeValues = [startDate, endDate];
-	const [range, setRange] = useState(defaultRangeValues);
+	const startDate: Date = moment(date.startDate).toDate() || PLUS_ONE_MONTH;
+	const endDate: Date = moment(date.endDate as moment.MomentInput).toDate() || PLUS_TWO_MONTHS;
+	const defaultRangeValues: [Date, Date] = [startDate, endDate];
+	const [range, setRange] = useState<[Date, Date]>(defaultRangeValues);
```

Comment:
Great job :taco:

Reply:
agreed...
 hey  uh...
  where'd you get that taco from?

Reply:
we had used tacos to praise peeps when they are doing great things on the previous project: https://www.heytaco.chat/